packages/features/bookings/components/event-meta/Members.tsx

```diff
@@ -38,40 +25,22 @@ export const EventMembers = ({ schedulingType, users, profile, entity }: EventMe
     !users.length ||
     (profile.name !== users[0].name && schedulingType === SchedulingType.COLLECTIVE);
 
-  const avatars: Avatar[] = shownUsers.map((user) => ({
-    title: `${user.name || user.username}`,
-    image: "image" in user ? `${user.image}` : `/${user.username}/avatar.png`,
-    alt: user.name || undefined,
-    href: `/${user.username}`,
-  }));
-
-  // Add organization avatar
-  if (entity.orgSlug) {
-    avatars.unshift({
-      title: `${entity.name}`,
-      image: `${WEBAPP_URL}/team/${entity.orgSlug}/avatar.png`,
-      alt: entity.name || undefined,
-      href: getOrgFullDomain(entity.orgSlug),
-    });
-  }
-
-  // Add profile later since we don't want to force creating an avatar for this if it doesn't exist.
-  avatars.unshift({
-    title: `${profile.name || profile.username}`,
-    image: "logo" in profile && profile.logo ? `${profile.logo}` : undefined,
-    alt: profile.name || undefined,
-    href: profile.username
-      ? `${CAL_URL}${pathname?.indexOf("/team/") !== -1 ? "/team" : ""}/${profile.username}`
-      : undefined,
-  });
-
-  const uniqueAvatars = avatars
-    .filter((item): item is AvatarWithRequiredImage => !!item.image)
-    .filter((item, index, self) => self.findIndex((t) => t.image === item.image) === index);
-
```

Comment:
I am not sure we were comparing images, it didn't make sense. We need to show all the users avatars and users should be unique already.